tools/src/main/scala/org/apache/spark/sql/rapids/tool/AppBase.scala

```diff
@@ -208,14 +208,22 @@ abstract class AppBase(
   }
 
   private val UDFRegex = ".*UDF.*"
-  private val UDFKeywords = Map(UDFRegex -> "UDF")
+
+  private val potentialIssuesRegexMap = Map(
+    UDFRegex -> "UDF", 
```

Comment:
I'm wondering if TimeAdd shows up in the plans or if it just shows as a +, I think its just a + with a cast so might not be easy to catch here.

Reply:
https://github.com/NVIDIA/spark-rapids/commit/7910229d025fab20081f54b28618f3db2be35225

After testing a few things, it doesn't shows up in the plan, I think it just shows up as +,  so would be hard to find.